community/es/build

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+patch -p1 < musl.patch
+sed -i 's|/doc/es\.1|/es\.1|g' Makefile.in
+
+./configure \
+    --prefix="$1"
```

Comment:
`--prefix="$1/usr"` should fix my comments about the directory locations.

Reply:
Done